.github/scripts/cygwin/build-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+source `dirname ${BASH_SOURCE[0]}`/../config.sh
+
+PATH="$TOOLCHAIN_PATH/bin:$TOOLCHAIN_PATH/lib/gcc/aarch64-pc-cygwin/:$TOOLCHAIN_PATH/lib/gcc/$TARGET/$GCC_VERSION/:$PATH"
```

Comment:
`$TARGET` variable can be used instead of `aarch64-pc-cygwin`. PS: I know the same mistake is in `.github/scripts/tests/execute-tests.sh`

Reply:
Changed